app/javascript/components/enrollment/steps/ExposureInformation.js

```diff
@@ -427,25 +494,47 @@ class ExposureInformation extends React.Component {
               type="switch"
```

Comment:
I don't love that this switch is disabled... at least I would think that toggling it off could clear all the options you currently have.  I feel like that should be a feature regardless (even if it's not done with the toggle) but the disabled toggle is confusing to me

Reply:
we've received feedback from the PO team to add a tooltip to this so users are not confused

Reply:
this is no longer relevant because the PO team would like to keep this fields separate, so it is now toggleable

Reply:
I'm still seeing the table displayed on the Patient page if the toggle is turned off, is this intentional?

Reply:
yep, the boolean field now is considered separate from the cohorts in the table

Reply:
in that case, shouldn't toggling the boolean hide the cohort table in enrollment, as well as in review and on the patient page?  because essentially it's turning off that the monitoree is a member of a cohort so it's confusing for the table to still display

Reply:
this is what the PO team has decided after much deliberation, I'm just implementing what they want, feel free to discuss with them though

Reply:
started convo here: https://mitrecorp.slack.com/archives/C029JM3PFC4/p1637239430103000

Reply:
ended up implementing the auto/manual toggle option described in the ticket